src/app/pages/movie/movie.component.html

```diff
@@ -0,0 +1,42 @@
+<div *ngIf="(pageDetails$ | async) as details" class="movie">
```

Comment:
Maybe we can create separate component for movie-details? Then template will be clear